### PR TITLE
feat: Increase map size in DrawBoundary

### PR DESCRIPTION
**Pull request by DafyddLlyr**
## What does this PR do?
 - Above the "md" breakpoint, increase the map size in the DrawBoundary component
 - Below this, keep things as is
 - The new width and height of 80vw x 80vh is chose quite arbitrarily - it seems to allow the map + instructions to appear on the screen at the same time. I expect this could be changed through feedback and UAT.

### Also...
- Tidy up styling, continue to migrate touched components from JSS to Emotion

image

https://user-images.githubusercontent.com/20502206/197004072-10393eb3-ef06-4339-9e5d-071243f92c98.mov

### CODE REVIEW

File: editor.planx.uk/src/@planx/components/DrawBoundary/Public/index.tsx [hunk 1]
```diff
@@ -21,28 +19,39 @@ import Upload, { FileUpload } from "./Upload";
...
-    padding: theme.spacing(1, 0),
+const MapContainer = styled(Box)(({ theme }) => ({
+  padding: theme.spacing(1, 0),
+  width: "100%",
+  height: "50vh",
+  [theme.breakpoints.up("md")]: {
+    height: "80vh",
+    width: "80vw",
+    position: "relative",
+    left: "calc(-40vw + 50%)",
```
Comment (DafyddLlyr) [inline editor.planx.uk/src/@planx/components/DrawBoundary/Public/index.tsx:30]: Here's the significant change ☝️

File: editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx [hunk 2]
```diff
@@ -180,6 +178,23 @@ function Component(props: Props) {
   }
 }
 
+const AutocompleteWrapper = styled(Box)(({ theme }) => ({
```
Comment (DafyddLlyr) [inline editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx:181]: Something worth noting with Mui v5 styles - this needed to be hoisted outside of `GetAddress` for it to work and render correctly.

File: editor.planx.uk/src/@planx/components/DrawBoundary/Public/index.tsx [hunk 3]
```diff
@@ -21,28 +19,39 @@ import Upload, { FileUpload } from "./Upload";
...
-    },
-    "& button:hover": {
-      backgroundColor: theme.palette.background.paper,
-    },
-    "& button:disabled": {
-      color: theme.palette.text.disabled,
-    },
+  "& my-map": {
+    width: "100%",
+    height: "100%",
```
Comment (DafyddLlyr) [inline editor.planx.uk/src/@planx/components/DrawBoundary/Public/index.tsx:34]: This allows us to drop `map.css`

File: editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx [hunk 4]
```diff
@@ -417,9 +408,9 @@ export function PropertyInformation(props: any) {
   return (
     <Card handleSubmit={formik.handleSubmit} isValid>
       <QuestionHeader title={title} description={description} />
-      <Box className={styles.map}>
+      <MapContainer>
         <p style={visuallyHidden}>
-          A static map centered on the property address, showing the Ordnance
+          A static map centred on the property address, showing the Ordnance
```
Comment (jessicamcinchak) [inline editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx:413]: Thank you :gb: :sweat_smile:

**Review by jessicamcinchak — APPROVED** [review]
So much space for drawing! :crayon: I'll be curious to hear user feedback about size on this one.

Definitely like taking this approach first, rather than right into openlayers built in full screen.

Thanks for so many nicely re-named boxes too :100:

**Comment by DafyddLlyr** [comment]
Closed in favour of https://github.com/theopensystemslab/planx-new/pull/1222 which has been passed forward for UAT

**Pull request closed without merging**